front-end/src/components/views/VideoPlayer.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+import '../../styles/videoPlayerStyles.css';
+import ReactPlayer from 'react-player';
+import { ApiUtil } from '../../lib/apiUtils';
+
+const VideoPlayer = ({ video, videoTimestamp, setVideoTimestamp }) => {
```

Comment:
videotimestamp is not used, why is it being passed?

Reply:
will be used on chart to video synching is implemented